packages/controller-utils/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
```

Comment:
I did a major release because I wasn't sure.  But I'm not actually aware of any breaking behavior between the old and new endpoints.

Reply:
Okay gotcha. I believe this is the second time this change has been marked as breaking in some way (either by noting it in the changelog or otherwise). I am okay with being on the safe side! So this looks good to me.

Reply:
Please add the `**BREAKING:**` prefix to that change entry though, so we know that this was intended. That's how we typically highlight which changes we are considering breaking.